docs/en/sql_reference/statements/grant.md

```diff
@@ -0,0 +1,376 @@
+---
+toc_priority: 39
+toc_title: GRANT
+---
+
+
+# GRANT
+
+- Grants [privileges](#grant-privileges) to ClickHouse user accounts or roles.
+- Assigns roles to user accounts or to another roles.
+
+To revoke privileges, use the [REVOKE](revoke.md) statement. Also you can list granted privileges by the [SHOW GRANTS](show.md#show-grants-statement) statement.
+
+## Granting Privilege Syntax {#grant-privigele-syntax}
+
+```sql
+GRANT [ON CLUSTER] privilege[(column_name [,...])] [,...] ON {db.table|db.*|*.*|table|*} TO {user | role | CURRENT_USER} [,...] [WITH GRANT OPTION]
+```
+- `privilege` — Type of privilege.
+- `role` — ClickHouse user role.
+- `user` — ClickHouse user account.
+
+The `WITH GRANT OPTION` clause sets [GRANT OPTION](#grant-option-privilege) privilege for `user` or `role`.
```

Comment:
There is no privilege named `GRANT OPTION`. We need just explain here that `WITH GRANT OPTION` means that grantee will able to grant passed privileges to someone else.